lib_comfyui/polling_client.py

```diff
@@ -20,11 +22,23 @@ class ComfyuiNodeWidgetRequests:
     @staticmethod
     def send(request_params):
         cls = ComfyuiNodeWidgetRequests
+        if cls.focused_webui_client_id is None:
+            return None
         clear_queue(cls.finished_comfyui_queue)
         webui_client_id = cls.focused_webui_client_id
         cls.last_params = request_params
         cls.loop.call_soon_threadsafe(cls.param_events[webui_client_id][cls.last_params['workflowType']].set)
-        return cls.finished_comfyui_queue.get()
+        
+        keep_waiting = True
+        while keep_waiting:
+            try:
+                result = cls.finished_comfyui_queue.get(timeout=1)
+                keep_waiting = False
+            except queue.Empty:
+                if getattr(shared_state, 'interrupted', False):
+                    return None
+
+        return result

```

Comment:
do you think this is fine considering that we might not only use this "send" function for sending image generation requests?